core/camel-api/src/main/java/org/apache/camel/ExchangePropertyKey.java

```diff
@@ -23,6 +23,7 @@
  */
 public enum ExchangePropertyKey {
 
+    ACTIVE_SPAN("OpenTracing.activeSpan"),
```

Comment:
Can you avoid doing changes to core in this PR and leave this where it was before.
Adding new to Exchange should be done in another PR

Reply:
I will revert this change.